crates/sui/tests/full_node_tests.rs

```diff
@@ -55,7 +56,7 @@ use tracing::info;
 #[sim_test]
 async fn test_full_node_follows_txes() -> Result<(), anyhow::Error> {
     let mut test_cluster = TestClusterBuilder::new().build().await?;
-    let node = test_cluster.start_fullnode().await?.sui_node;
+    let fullnode = test_cluster.start_fullnode().await.sui_node;
```

Comment:
do you think it would be easy to have fullnodes started this way run in their own simnodes?

Reply:
It should already be the case?

Reply:
ok, i see it now, sorry